cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -205,6 +208,9 @@ type PostRequestBody struct {
 	from      *mail.Address
 	replyTo   *mail.Address
 	publicKey crypto.PublicKey
+	// encryption method to use
+	// required: false
```

Comment:
Can you add a bit of documentation about what are accepted values so that the API documentation has this information.